scripts/tools/release-cli/release-cli.js

```diff
@@ -90,6 +92,22 @@ function getNextVersion() {
   return semver.inc(getVersion('core'), 'patch');
 }
 
+function getVersionSuffix() {
```

Comment:
good idea extracting this into a function 👍 

Could you add some JSDoc on the function header. Would be helpful to know what it returns and what the returned value format looks like without having to read the whole code.